<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgWorkpiece</class>
 <widget class="QDialog" name="DlgWorkpiece">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>225</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>工作计数清零</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res.qrc">
    <normaloff>:/images/image/logo.png</normaloff>:/images/image/logo.png</iconset>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="box_bg_jubaonumber">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>拒曝总数</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="box_dp_baoguang">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>底片曝光</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_number_tip">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>提示：</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="box_dp_clear">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>底片清洁</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="box_bg_totalnumber">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>完成总数</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_dp_clear">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>(0)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_total_number">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>30</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>(0)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_dp_baoguang">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>(0)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_jubaonumber">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>70</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>(0)</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
